schemaless/sources.py

```diff
@@ -70,9 +126,26 @@ def yield_records(self):
             reader = DictReader(inf)
 
             for line in reader:
-                yield {self.FIELDS[key]: val.strip()
-                       for key, val in line.items()
-                       if key in self.FIELDS and val}
+                calc_fields = {}
+                ret = {}
+                for line_key, ret_key in self.FIELDS.items():
+                    # save calculated Fields for later
+                    if isinstance(ret_key, Field):
+                        calc_fields[line_key] = ret_key
+                        continue
+
+                    if line_key not in line:
+                        continue
+
+                    val = (line.get(line_key) or "").strip()
+                    if val:
+                        ret[ret_key] = val
+
+                for key, field in calc_fields.items():
```

Comment:
This means that the normalized addresses gets written to the schemaless file right? Did we want to start including fields we've done our own processing on in the schemaless file (thought we weren't doing that)?

Reply:
Yep, it gets written out.

> Did we want to start including fields we've done our own processing on in the schemaless file (thought we weren't doing that)?

I didn't think about that. I guess there's no reason to write it out, but is there a strong reason _not_ to?

Reply:
Approving, as discussed offline, we just won't write the normalized address to the schemelass for now

Reply:
Changed to not write out computed fields in 627caddee44cd473e810eaeb41fe9eeb8099b8b5